#' pop.STR allele frequencies
#'
#' Data was generated by selecting all possible markers and the corresponding
#' population (meta-)group in the pop.STR tool.
#'
#' @source \url{http://spsmart.cesga.es/popstr.php}
#' @source Amigo J, Phillips C, Salas T, Fernández Formoso L, Carracedo A, Lareu
#'   M. pop.STR - An online population frequency browser for established and new
#'   forensic STRs. Forensic Sci. Int. Gene. Suppl. 2009
#'   (10.1016/j.fsigss.2009.08.178)
#' @name data_popstr

#' @rdname data_popstr
"ft_popstr_europe"

#' @rdname data_popstr
"ft_popstr_nw_spain"

#' @rdname data_popstr
"ft_popstr_israel_carmel_druze"

#' @rdname data_popstr
"ft_popstr_cambodia_cambodian"

#' @rdname data_popstr
"ft_popstr_china_dai"

#' @rdname data_popstr
"ft_popstr_china_daur"

#' @rdname data_popstr
"ft_popstr_china_han"

#' @rdname data_popstr
"ft_popstr_japan_japanese"

#' @rdname data_popstr
"ft_popstr_siberia_yakut"

#' @rdname data_popstr
"ft_popstr_central_african_republic_biaka_pygmies"

#' @rdname data_popstr
"ft_popstr_d_r_congo_mbuti_pygmies"

#' @rdname data_popstr
"ft_popstr_kenya_bantu_ne"

#' @rdname data_popstr
"ft_popstr_namibia_san"

#' @rdname data_popstr
"ft_popstr_nigeria_yoruba"

#' @rdname data_popstr
"ft_popstr_senegal_mandenka"

#' @rdname data_popstr
"ft_popstr_somalia"

#' @rdname data_popstr
"ft_popstr_south_africa_bantu"

#' @rdname data_popstr
"ft_popstr_brazil_karitiana"

#' @rdname data_popstr
"ft_popstr_brazil_surui"

#' @rdname data_popstr
"ft_popstr_colombia_colombian"

#' @rdname data_popstr
"ft_popstr_dominican_republic"

#' @rdname data_popstr
"ft_popstr_mexico_maya"

#' @rdname data_popstr
"ft_popstr_mexico_pima"

#' @rdname data_popstr
"ft_popstr_israel_central_palestinian"

#' @rdname data_popstr
"ft_popstr_israel_negev_bedouin"

#' @rdname data_popstr
"ft_popstr_bougainville_nan_melanesian"

#' @rdname data_popstr
"ft_popstr_new_guinea_papuan"

#' @rdname data_popstr
"ft_popstr_china_uygur"

#' @rdname data_popstr
"ft_popstr_pakistan_balochi"

#' @rdname data_popstr
"ft_popstr_pakistan_burusho"

#' @rdname data_popstr
"ft_popstr_pakistan_hazara"

#' @rdname data_popstr
"ft_popstr_pakistan_kalash"

#' @rdname data_popstr
"ft_popstr_pakistan_makrani"

#' @rdname data_popstr
"ft_popstr_pakistan_pathan"

#' @rdname data_popstr
"ft_popstr_pakistan_sindhi"

#' @rdname data_popstr
"ft_popstr_algeria_mzab_mozabite"

#' @rdname data_popstr
"ft_popstr_china_hezhen"

#' @rdname data_popstr
"ft_popstr_china_lahu"

#' @rdname data_popstr
"ft_popstr_china_miaozu"

#' @rdname data_popstr
"ft_popstr_china_mongola"

#' @rdname data_popstr
"ft_popstr_china_naxi"

#' @rdname data_popstr
"ft_popstr_china_oroqen"

#' @rdname data_popstr
"ft_popstr_china_she"

#' @rdname data_popstr
"ft_popstr_china_tu"

#' @rdname data_popstr
"ft_popstr_china_tujia"

#' @rdname data_popstr
"ft_popstr_china_xibo"

#' @rdname data_popstr
"ft_popstr_china_yizu"


#' NIST: US Allele frequencies
#'
#' Data extracted from the NIST 1036 Revised U.S. Population Dataset (July
#' 2017). The following collections are available:
#'
#' \describe{
#'
#' \item{ft_nist_all}{All data (average over the 1036 individuals)}
#'
#' \item{ft_nist_asian}{Asian}
#'
#' \item{ft_nist_african_american}{African American}
#'
#' \item{ft_nist_caucasian}{Caucasian}
#'
#' \item{ft_nist_hispanic}{Hispanic}
#'
#' }
#'
#' @source \url{https://strbase.nist.gov/NISTpop.htm}
#' @source Hill, C. R., Duewer, D. L., Kline, M. C., Coble, M. D., & Butler, J.
#'   M. (2013). US population data for 29 autosomal STR loci. Forensic Science
#'   International: Genetics, 7(3), e82-e83.
#'   DOI:https://doi.org/10.1016/j.fsigen.2012.12.004
#' @name data_nist


#' @rdname data_nist
"ft_nist_caucasian"

#' @rdname data_nist
"ft_nist_asian"

#' @rdname data_nist
"ft_nist_african_american"

#' @rdname data_nist
"ft_nist_hispanic"

#' @rdname data_nist
"ft_nist_all"


#' STRidER 2.0
#'
#' As per the STRidER website, be advised that in these tables, 1 represents all
#' rare alleles shorter than the accepted allele categories. The value 99
#' represents all rare alleles longer than the accepted categories.
#'
#' @source \url{https://strider.online/frequencies}
#' @source Bodner M., Bastisch, I., Butler, J.M., Fimmers, R., Gill, P., Gusmão,
#'   L., Morling, N., Phillips, C., Prinz, M., Schneider, P.M., Parson, W.
#'   (2016), 'Recommendations of the DNA Commission of the International Society
#'   for Forensic Genetics (ISFG) on quality control of autosomal Short Tandem
#'   Repeat allele frequency databasing (STRidER).', Forensic Sci. Int. Genet.
#'   24, 97-102
#' @name data_strider

#' @rdname data_strider
"ft_strider_austria"

#' @rdname data_strider
"ft_strider_belgium"

#' @rdname data_strider
"ft_strider_bosnia_herzegowina"

#' @rdname data_strider
"ft_strider_czech_republic"

#' @rdname data_strider
"ft_strider_denmark"

#' @rdname data_strider
"ft_strider_finland"

#' @rdname data_strider
"ft_strider_france"

#' @rdname data_strider
"ft_strider_germany"

#' @rdname data_strider
"ft_strider_greece"

#' @rdname data_strider
"ft_strider_hungary"

#' @rdname data_strider
"ft_strider_ireland"

#' @rdname data_strider
"ft_strider_montenegro"

#' @rdname data_strider
"ft_strider_norway"

#' @rdname data_strider
"ft_strider_poland"

#' @rdname data_strider
"ft_strider_slovakia"

#' @rdname data_strider
"ft_strider_slovenia"

#' @rdname data_strider
"ft_strider_spain"

#' @rdname data_strider
"ft_strider_sweden"

#' @rdname data_strider
"ft_strider_switzerland"


#' Ferdous 2010: Bangladesh
#'
#' Allele frequencies of ten autosomal STR loci, D3S1358, vWA, D16S539, D2S1338,
#' D8S1179, D21S11, D18S51, D19S433, TH01, and FGA were investigated in Chakma
#' and Tripura tribal populations of Bangladesh. In both the populations, all
#' loci were in Hardy-Weinberg equilibrium except for FGA locus in Chakma and
#' D21S11 in Tripura. All the loci were highly polymorphic in Chakma population
#' with an observed heterozygosity  of 0.7 and moderately polymorphic in Tripura
#' population . However, both the population showed least polymorphism at TH01
#' locus . A comparison between Chakma and Tripura population data revealed
#' statistically significant differences in allele frequency distribution for
#' most of the loci. A similar comparison with the mainstream Bengali population
#' using previously published data from this lab also showed significant
#' difference in allele frequency with these two tribal populations.
#'
#' @source Ferdous, A., Ali, M. E., Alam, S., Hasan, M., Hossain, T., &
#'   Akhteruzzaman, S. (2010). Allele Frequencies of 10 Autosomal STR Loci from
#'   Chakma and Tripura Tribal Populations in Bangladesh. Molecular biology
#'   international, 2010.
#'
#' @name ferbous2010

#' @rdname ferbous2010
"ft_ferdous2010_bangladesh_chakma"

#' @rdname ferbous2010
"ft_ferdous2010_bangladesh_tripura"

#' Zeyad 2020: United Arab Emirates Y-STRs
#'
#' The recently introduced 6-dyes Yfiler® Plus multiplex kit which includes 27
#' Y-STR loci has been used to study 436 UAE Arab male individuals. In addition
#' to the 17 Y-STRs traditionally used in Yfiler® multiplex kit, Yfiler® Plus
#' has ten additional Y-STRs. It includes seven rapidly mutating Y-STRs (RM
#' Y-STRs) namely DYS560, DYS576, DYS627, DYS518, DYS449 and DYF387S1ab, and
#' three moderately mutating Y-STRs namely DYS481, DYS533 and DYS460. The RM
#' Y-STRs were already proven to be powerful for discriminating closely related
#' and unrelated males’ individuals. As a result, a total number of 419
#' haplotypes were observed out of 436 samples analysed of UAE Arab males’
#' population, whereas 15 haplotypes were shared between two individuals and one
#' haplotypes were shared between three individuals. The discrimination capacity
#' (DC) for the 27 loci was found to be 0.9618 whereas haplotype diversity (HD)
#' was found to be 0.99982. AMOVA results showed that UAE population clustered
#' along with Saudi Arabia East [Arab] population which indicates that they have
#' a closer genetic relation. While it was standing away compared to other
#' relevant populations. In conclusion, Yfiler® Plus multiplex kit showed higher
#' power of discrimination than all previous studies conducted on the same
#' population. This study considered the first study conducted using 27 Y-STR
#' loci in this population. Geographic distribution of the populations analysed
#' in this study are well reflected in the genetic distance results.
#'
#' @source Zeyad, T., Adam, A., Alghafri, R., & Iratni, R. (2020). Study of 27
#'   Y-STR markers in United Arab Emirates population. Forensic Science
#'   International: Reports, 100057.
"ft_zeyad2020_uae_ystrs"
